sdcm/sct_config.py

```diff
@@ -1091,6 +1091,12 @@ class SCTConfiguration(dict):
         dict(name="scylla_mgmt_upgrade_to_repo", env="SCT_SCYLLA_MGMT_UPGRADE_TO_REPO", type=str,
              help="Url to the repo of scylla manager version to upgrade to for management tests"),
 
+        dict(name="mgmt_restore_params", env="SCT_MGMT_RESTORE_PARAMS", type=dict_or_str,
```

Comment:
those apply only to manager tests ? or also to the nemesis that does restore ?

Reply:
only Manager tests